### ISSUE TITLE
Style/MultilineBlockLayout wrongly reports subclass declaration

### ISSUE THREAD

**Issue by ojab** [rubocop/rubocop#1224]
Testcase:

``` ruby
class Foo < Bar
  lambda { joins
           .distinct }
end
```

rubocop-master reports:

```
$ rubocop -D --only Style/MultilineBlockLayout /tmp/1.rb
Inspecting 1 file
C

Offenses:

/tmp/1.rb:1:12: C: Style/MultilineBlockLayout: expression at 2, 12 is on the same line as the block start
class Foo < Bar
           ^

1 file inspected, 1 offense detected
```
